spec/dish_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'dish'
+
+chicken_chow_mein = Dish.new('Chicken Chow Mein', 5.95)
```

Comment:
Even better, use the [explicit `subject`](https://www.relishapp.com/rspec/rspec-core/docs/subject/explicit-subject) RSpec pattern.